packages/visx-xychart/src/types/series.ts

```diff
@@ -18,7 +18,7 @@ export type EventHandlerParams<Datum> = {
   /** Coordinates of the event in svg space. */
   svgPoint?: { x: number; y: number };
   /** The PointerEvent or FocusEvent. */
-  event: PointerEvent | FocusEvent;
+  event?: PointerEvent | FocusEvent;
```

Comment:
added to the PR description. you had a commit message with it, but just want to make it lands in the 1.4 changelog as an enhancement

Reply:
good catch, thanks 🙏